### PR TITLE
OSASINFRA-3132: Add triage reminders

### PR DESCRIPTION
**Pull request by stephenfin**
This is mostly duplicated from the pretriage script, but it gives us the ability to remind triagers to check their assigned bugs.

### CODE REVIEW

File: cmd/triage/notify.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+)
+
+func notify(slackHook string, slackClient *http.Client, issue jira.Issue, assignee TeamMember) error {
+	var msg bytes.Buffer
+	err := json.NewEncoder(&msg).Encode(struct {
+		LinkNames bool   `json:"link_names"`
+		Text      string `json:"text"`
+	}{
+		LinkNames: true,
+		Text:      "<@" + assignee.SlackId + "> you have been assigned the triage of this bug: " + jiraBaseURL + "browse/" + issue.Key,
```
Comment (mandre) [inline cmd/triage/notify.go:20]: Do you think we should differentiate from the pre-triage message to highlight this is a reminder?

Comment (stephenfin) [reply cmd/triage/notify.go:20]: I thought about that, but couldn't come up with a better message. Any suggestions?

**Comment by stephenfin** [comment]
My one outstanding query on this is whether we want to post a message per bug, or count the total number of untriaged bugs per assignee and post a notification about that? Doing the latter means we need to process all issues in one go, as opposed to one by one like we do now. @mandre any thoughts?

File: cmd/triage/notify.go (hunk 1 above)
Comment (mandre) [reply cmd/triage/notify.go:20]: What do you think of "@slack_user gentle reminder to triage these bugs: list of bugs"?

Comment (stephenfin) [reply cmd/triage/notify.go:20]: Done

**Comment by stephenfin** [comment]
> > My one outstanding query on this is whether we want to post a message per bug, or count the total number of untriaged bugs per assignee and post a notification about that? Doing the latter means we need to process all issues in one go, as opposed to one by one like we do now. @mandre any thoughts?
> 
> Having a single notification when a user has multiple untriaged bugs would be nice, and similar to what we're already doing for the doctext notification.

Done :+1:

**Comment by pierreprinetti** [comment]
@stephenfin This needs a 
```shell
go fmt ./...
```

**Comment by stephenfin** [comment]
> @stephenfin This needs a
> 
> ```shell
> go fmt ./...
> ```

Missed that. Done.

**Pull request merged**